apps/nextjs-website/src/components/organisms/Auth/AccountActivatedCard.tsx

```diff
@@ -9,6 +9,7 @@ const AccountActivatedCard = () => {
     auth: { accountActivated },
   } = translations;
 
+  // TODO: refactor using SingleCard component
```

Comment:
this is TODO is relative to this issue https://pagopa.atlassian.net/browse/DEV-1178